2021/01/14 14:00:33 : Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 665, in storage_data
    getattr(self.get_source_cart(), 'finish_storage_data')()
  File "/home/hoang/Desktop/v32/cartmigration/models/basecart.py", line 3929, in finish_storage_data
    folder_upload = self.get_folder_upload(self._migration_id)
  File "/home/hoang/Desktop/v32/cartmigration/models/basecart.py", line 3941, in get_folder_upload
    folder_upload = root_path + '/' + DIR_UPLOAD + '/' + get_config_ini('src', 'file', file = 'test.ini')
TypeError: must be str, not NoneType

2021/01/14 14:01:00 : orders_errors 1: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4778, in order_import
    'product_qty': convert['items']['qty'],
TypeError: list indices must be integers or slices, not str

2021/01/14 14:01:08 : orders_errors 2: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4774, in order_import
    'product_id': product_id,
UnboundLocalError: local variable 'product_id' referenced before assignment

2021/01/14 14:01:17 : orders_errors 3: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4774, in order_import
    'product_id': product_id,
UnboundLocalError: local variable 'product_id' referenced before assignment

2021/01/14 14:01:27 : orders_errors 4: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4774, in order_import
    'product_id': product_id,
UnboundLocalError: local variable 'product_id' referenced before assignment

2021/01/14 14:01:46 : orders_errors 5: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4774, in order_import
    'product_id': product_id,
UnboundLocalError: local variable 'product_id' referenced before assignment

2021/01/14 14:01:55 : orders_errors 6: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4774, in order_import
    'product_id': product_id,
UnboundLocalError: local variable 'product_id' referenced before assignment

2021/01/14 14:02:09 : orders_errors 7: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4774, in order_import
    'product_id': product_id,
UnboundLocalError: local variable 'product_id' referenced before assignment

2021/01/14 14:02:21 : orders_errors 8: Traceback (most recent call last):
  File "/home/hoang/Desktop/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/home/hoang/Desktop/v32/cartmigration/models/cart/woocommerce.py", line 4774, in order_import
    'product_id': product_id,
UnboundLocalError: local variable 'product_id' referenced before assignment

